content/en/docs/tutorials/security/apparmor.md

```diff
@@ -122,8 +122,7 @@ gke-test-default-pool-239f5d02-xwux: kubelet is posting ready status. AppArmor e
 
 {{< note >}}
 AppArmor is currently in beta, so options are specified as annotations. Once support graduates to
-general availability, the annotations will be replaced with first-class fields (more details in
-[Upgrade path to GA](#upgrade-path-to-general-availability)).
+general availability (GA), the annotations will be replaced with first-class fields.
```

Comment:
Hello @kadtendulkar .
Does it make sense to either remove or clean up the note?
I suggested a brief sentence removing the rest of the note.